fboss/platform/configs/tahan800bc/sensor_service.json

```diff
@@ -1441,507 +915,2659 @@
-          },
-          "compute": "@/1000.0"
+            "upperCriticalVal": 12100,
+            "maxAlarmVal": 11000,
+            "minAlarmVal": 2400,
+            "lowerCriticalVal": 1250
+          }
         },
         {
-          "name": "COMe_PU32_TDA38640_P1V05_STBY_VOUT",
-          "sysfsPath": "/run/devmap/sensors/COME_PU32_TDA38640/in2_input",
-          "type": 1,
+          "name": "FCB_FANTRAY2_FAN6_INPUT",
+          "sysfsPath": "/run/devmap/sensors/SMB_BCB_FAN_CPLD_2/fan6_input",
+          "type": 4,
           "thresholds": {
-            "upperCriticalVal": 1.46,
-            "lowerCriticalVal": 0.66
-          },
-          "compute": "@/1000.0"
+            "upperCriticalVal": 14850,
+            "maxAlarmVal": 13500,
+            "minAlarmVal": 2900,
+            "lowerCriticalVal": 1500
+          }
         },
         {
-          "name": "COMe_PU32_TDA38640_P1V05_STBY_TEMP",
-          "sysfsPath": "/run/devmap/sensors/COME_PU32_TDA38640/temp1_input",
-          "type": 3,
+          "name": "FCB_FANTRAY2_FAN7_INPUT",
+          "sysfsPath": "/run/devmap/sensors/SMB_BCB_FAN_CPLD_2/fan7_input",
+          "type": 4,
           "thresholds": {
-            "maxAlarmVal": 100,
-            "upperCriticalVal": 115
-          },
-          "compute": "@/1000.0"
-        },
-        {
-          "name": "COMe_PU32_TDA38640_P1V05_STBY_PIN",
-          "sysfsPath": "/run/devmap/sensors/COME_PU32_TDA38640/power1_input",
-          "type": 0,
-          "compute": "@/1000000.0"
-        },
-        {
-          "name": "COMe_PU32_TDA38640_P1V05_STBY_POUT",
-          "sysfsPath": "/run/devmap/sensors/COME_PU32_TDA38640/power2_input",
-          "type": 0,
-          "compute": "@/1000000.0"
-        },
-        {
-          "name": "COMe_PU32_TDA38640_P1V05_STBY_IIN",
-          "sysfsPath": "/run/devmap/sensors/COME_PU32_TDA38640/curr1_input",
-          "type": 2,
-          "compute": "@/1000.0"
+            "upperCriticalVal": 12100,
+            "maxAlarmVal": 11000,
+            "minAlarmVal": 2400,
+            "lowerCriticalVal": 1250
+          }
         },
         {
-          "name": "COMe_PU32_TDA38640_P1V05_STBY_IOUT",
-          "sysfsPath": "/run/devmap/sensors/COME_PU32_TDA38640/curr2_input",
-          "type": 2,
+          "name": "FCB_FANTRAY2_FAN8_INPUT",
+          "sysfsPath": "/run/devmap/sensors/SMB_BCB_FAN_CPLD_2/fan8_input",
+          "type": 4,
           "thresholds": {
-            "upperCriticalVal": 20
-          },
-          "compute": "@/1000.0"
-        },
+            "upperCriticalVal": 14850,
+            "maxAlarmVal": 13500,
+            "minAlarmVal": 2900,
+            "lowerCriticalVal": 1500
+          }
+        }
+      ]
+    },
+    {
+      "slotPath": "/SMB_FRU_SLOT@0",
+      "pmUnitName": "SMB",
+      "sensors": [],
+      "versionedSensors": [
+        {
+          "sensors": [
+            {
+              "name": "TH5_VDD_CORE_IOUT",
```

Comment:
Sensors like "TH5_VDD_CORE_IOUT" and "TH5_VDD_CORE_VIN" have identical definitions across all "versionedSensors" field. 

Please identify all the other sensors like them and move them to "sensors" field in line 1142.

Reply:
Hi kimdo

1.Thanks for your reply, i know your point, in fact "TH5_VDD_CORE_IOUT" and "TH5_VDD_CORE_VIN"  are versioned sensors so i put it in SMB_FRU_SLOT. The SMB common sensor we put in the TAHAN slot.

2.All the number combinations(productionstate, productversion, product subversion) are as follow(in the future PVT will add new)
![image](https://github.com/user-attachments/assets/07254f91-78b0-43f9-b9b3-c4c911487583)

3.So i think every number combination will have these sensors according to current platform config structure. If i am wrong please correct me, than you!